src/contracts/core/DelegationManager.sol

```diff
@@ -510,199 +505,178 @@ contract DelegationManager is
+        address operator,
+        address staker,
+        IStrategy strategy,
+        uint256 existingPrincipalShares,
+        uint256 addedShares,
+        uint64 totalMagnitude
+    ) internal {
+        _updateDepositScalingFactor({
+            staker: staker,
+            strategy: strategy,
+            existingPrincipalShares: existingPrincipalShares,
+            addedShares: addedShares,
+            totalMagnitude: totalMagnitude
+        });
+
+        // based on total magnitude, update operators delegatedShares
+        uint256 delegatedShares = DelegatedShares.unwrap(WithdrawableShares.wrap(addedShares).toDelegatedShares(totalMagnitude));
+        operatorDelegatedShares[operator][strategy] += delegatedShares;
+
+        // TODO: What to do about event wrt scaling?
+        emit OperatorSharesIncreased(operator, staker, strategy, delegatedShares);
     }
 
-    // @notice Decreases `operator`s delegated shares in `strategy` by `shares` and emits an `OperatorSharesDecreased` event
-    function _decreaseOperatorShares(address operator, address staker, IStrategy strategy, uint256 shares) internal {
-        // This will revert on underflow, so no check needed
-        operatorShares[operator][strategy] -= shares;
-        emit OperatorSharesDecreased(operator, staker, strategy, shares);
+    /**
+     * @notice Decreases `operator`s delegated delegatedShares in `strategy` based on staker's removed shares and operator's totalMagnitude
+     * @param operator The operator to decrease the delegated delegated shares for
+     * @param staker The staker to decrease the delegated delegated shares for
+     * @param strategy The strategy to decrease the delegated delegated shares for
+     * @param delegatedShares The delegatedShares to remove from the operator's delegated shares
+     */
+    function _decreaseDelegation(
+        address operator,
+        address staker,
+        IStrategy strategy,
+        uint256 delegatedShares
+    ) internal {
+        // based on total magnitude, decrement operator's delegatedShares
+        operatorDelegatedShares[operator][strategy] -= delegatedShares;
+        // TODO: What to do about event wrt scaling?
+        emit OperatorSharesDecreased(operator, staker, strategy, delegatedShares);
     }
 
     /**
-     * @notice Removes `shares` in `strategies` from `staker` who is currently delegated to `operator` and queues a withdrawal to the `withdrawer`.
+     * @notice Removes `sharesToWithdraw` in `strategies` from `staker` who is currently delegated to `operator` and queues a withdrawal to the `withdrawer`.
      * @dev If the `operator` is indeed an operator, then the operator's delegated shares in the `strategies` are also decreased appropriately.
-     * @dev If `withdrawer` is not the same address as `staker`, then thirdPartyTransfersForbidden[strategy] must be set to false in the StrategyManager.
+     * @dev If `withdrawer` is not the same address as `staker`
```

Comment:
just delete this comment if this functionality doesn't exist anymore. it's confusing to have the sentence just drop off